rebar.config

```diff
@@ -34,23 +29,120 @@
   {seestar, ".*", {git, "git://github.com/arcusfelis/seestar.git", "e9700e1e7e8226173ad76d22be9d3a4e0ef77310"}},
 
   {cache_tab, ".*", {git, "git://github.com/processone/cache_tab", {tag, "1.0.4"}}},
-  {stringprep, ".*", {git, "git://github.com/processone/stringprep.git", {tag, "1.0.6"}}},
+  {stringprep, ".*", {git, "git://github.com/erszcz/stringprep.git", {branch, "list-deps"}}},
```

Comment:
Good catch, it was merged and I remember changing the dep to mainstream... Probably a rebase reverted it without me noticing it. I'll fix that.